dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -7,7 +7,7 @@ import {
 } from '@guardian/source/foundations';
 import { Hide, Link } from '@guardian/source/react-components';
 import { ArticleDesign, type ArticleFormat } from '../../lib/articleFormat';
-import { isMediaCard } from '../../lib/cardHelpers';
+import { isMediaCard as isAMediaCard } from '../../lib/cardHelpers';
```

Comment:
Yeah, think I prefer this to what you had earlier (where `isAMediaCard` was the const declared on line 456) 👍